d1985r3.md

```diff
@@ -65,7 +65,7 @@ parameters:
 ```cpp
 template <template <typename> typename F>
 using H = F<int>;
-apply<H, G> // error, can't pass H as arg1 of apply, and G as arg2
+using t1 = apply<H, G> // error, can't pass H as arg1 of apply, and G as arg2
```

Comment:
I don't know if we need to have "runnable" examples. If there is a missing ; here.